chia/cmds/wallet.py

```diff
@@ -99,18 +99,19 @@ def show_cmd(wallet_rpc_port: int, fingerprint: int) -> None:
     asyncio.run(execute_with_wallet(wallet_rpc_port, fingerprint, {}, print_balances))
 
 
-@wallet_cmd.command("get_address", short_help="Get a wallet receive address")
+@wallet_cmd.command("get_address", short_help="Get the current public wallet address")
```

Comment:
What is "the current public wallet address" and how is it both what you would get with `--new_address` and without?

Reply:
Thats a good idea

Reply:
I'm not sure if I expressed myself clearly.  "Get the current public wallet address" to me describes a single functionality to get a specific address.  The new feature being added in this PR seems to give the choice between two addresses.  Either the 'current' one or the 'next' one.  "Get a wallet receive address" seems more accurate for the intended behavior of this PR.